platforms/hyperledger-besu/configuration/roles/create/helm_component/templates/node_besu.tpl

```diff
@@ -61,6 +61,9 @@ spec:
         rpc: {{ peer.rpc.port }}
         ws: {{ peer.ws.port }}
 
+    permissioning:
```

Comment:
This should also logically go within the node section itself

Reply:
updated